spec/system/partner_system_spec.rb

```diff
@@ -112,6 +113,34 @@
       end
     end
 
+    describe 'invite and approve partner if organization opts in single step invite and approve' do
+      let!(:partner_uninvited) { create(:partner, status: 'uninvited', name: Faker::Name.name, email: Faker::Internet.email) }
+      
+      context 'when single-step inviting and approving a partner successfully' do
+        before do
+          @organization.single_step_invite_and_approve = true
+          @organization.save!
+
+          partner_uninvited.primary_user.delete
+          expect(partner_uninvited.status).not_to eq(:invited)
```

Comment:
This isn't what `before` blocks are for - that's for setting up data. There shouldn't be any `expect` statements in it.